crates/ruff_cli/src/commands/run.rs

```diff
@@ -128,7 +132,8 @@ pub fn run(
             acc += item;
             acc
         });
-
+    // check imports?
```

Comment:
Is this a todo?

Reply:
Yes it is, I'll edit the comment appropriately